pyproject.toml

```diff
@@ -53,9 +57,29 @@ source = "vcs"
 version-file = "src/GOSTurban/_version.py"
 
 [tool.codespell]
-skip = 'docs/_build,docs/references.bib,__pycache__,*.png,*.gz,*.whl'
+skip = 'docs/_build,docs/references.bib,__pycache__,*.png,*.gz,*.whl,pyproject.toml,*csv,notebooks/Implementations/WSF_DECAT_B_ExploringIDCerrors.ipynb,notebooks/Implementations/URB_SEAU1_NovelUrbanization/MAP_Urbanization.ipynb'
 ignore-regex = '^\s*"image\/png":\s.*'
-ignore-words-list = "gost,"
+ignore-words-list = "gost,GOST,ALOS,ans,NAM"
 
 [tool.ruff.lint.pydocstyle]
 convention = "numpy"
+
+[tool.ruff]
+exclude = [
+	"WSF_DECAT_B_ExploringIDCerrors.ipynb",
+	"KAZ_Urbanization_Review.ipynb",
+	"Urban_metrics_Fullness.ipynb",
+	"Urban_metrics_Shape.ipynb",
+	"Urban_metrics_Sprawl.ipynb",
+	"Urban_metrics_Structure.ipynb",
+	"URB_DECAT_B_ExploringGHSSMODcode.ipynb",
+	"URB_SCAUR_UKR_B_I_urbanizationReview.ipynb",
+	"URB_SEAU1_B_A_Ka_ExtractDataUrban.ipynb",
+	"Zonal_statistics.ipynb",
+	"Data_Preparation.ipynb",
+	"URB_SEAU1_B_A_Ka_NovelUrbanizaton.ipynb",
+	"Create_Mosaick_Datasets.ipynb",
+	"GHSL_Standardize_To_Country.ipynb",
+	"MAP_Urbanization.ipynb",
+	"NGA_specific_results.ipynb",
+]
```

Comment:
Many of the notebooks in the `Implementations/` folder contain paths to local file systems, or reference source code held relative to some local file system, so these don't work. There are also many examples of references to variables that are not used. I didn't want to delete these notebooks in the event they are of some importance.

Please weigh in and let me know what you think @bpstewar

Reply:
@g4brielvs looking for some guidance here as well beyond the overall review.

Reply:
@bpstewar @andresfchamorro